bigcache.go

```diff
@@ -40,12 +41,19 @@ const (
 	Deleted = RemoveReason(3)
 )
 
+// New initialize new instance of BigCache
+func New(ctx context.Context, config Config) (*BigCache, error) {
+	return newBigCache(ctx, config, &systemClock{})
+}
+
 // NewBigCache initialize new instance of BigCache
+// deprecated, please use New(ctx, config) instead, New takes in context and can gracefully
+// shutdown with context cancellations
```

Comment:
```suggestion
// NewBigCache initialize new instance of BigCache
//
// Deprecated: NewBigCache is deprecated, please use New(ctx, config) instead, New takes in context and can gracefully
// shutdown with context cancellations
```
https://github.com/golang/go/wiki/Deprecated

Reply:
Fixed